h5p-accordion.js

```diff
@@ -82,28 +83,85 @@ H5P.Accordion = (function ($) {
     var self = this;
     var titleId = 'h5p-panel-link-' + this.idPrefix + id;
     var contentId = 'h5p-panel-content-' + self.idPrefix + id;
+    var expandCollapseOption = this.params.expandCollapseOption;
+    var multipleAccordionsOpen = this.params.multipleAccordionsOpen;
+    var titleAriaExpanded, titleAriaExpandedFirst, titlePanelExpanded, titlePanelExpandedFirst,
+      panelAriaHidden, panelAriaHiddenFirst, panelStyleDisplay, panelStyleDisplayFirst;
 
     var toggleCollapse = function () {
-      if (self.$expandedTitle === undefined || !self.$expandedTitle.is($title)) {
-        self.collapseExpandedPanels();
-        self.expandPanel($title, $content);
+      // Check if the panel is already expanded.
+      if ($(this).hasClass("h5p-panel-expanded")) {
+        // It is expanded, so collapse it.
+        self.collapsePanel($title, $content);
       }
+      // The title that was clicked isn't expanded.
       else {
-        self.collapsePanel($title, $content);
+        // Check if you should close all other panels before opening this one.
+        if (multipleAccordionsOpen === 'openOne') {
+          // Since the first panel is already open, and this is the first time through, you need to make sure $expandedTitle & $expandedPanel are set.
+          if (loadFirstPanel && expandCollapseOption === "expandedFirstOnly") {
+            self.$expandedTitle = $('#h5p-panel-link-0-0'); // Get the first panel title.
+            self.$expandedPanel = $('#h5p-panel-content-0-0'); // Get the first panel.
```

Comment:
I'm assuming there are content types that could have 2 (or more) accordions as part of a larger content type. Is that correct?

A little tweak to the code so that it will return a jQuery object instead of html.
`self.$expandedTitle = $(self.elements).eq(0);`
`self.$expandedPanel = $(self.elements).eq(1);`

Thoughts?